exporter/keystoneexporter/keystone.go

```diff
@@ -0,0 +1,130 @@
+package keystoneexporter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	"github.com/google/uuid"
+)
+
+var (
+	NilKsEvent   = KsEvent{}
+	NilKsMessage = KsMessage{}
+	hostname     = ""
+	ksClient *http.Client     = nil
+	ksGatewayURL = ""
+)
+
+func init() {
+	hname, err := os.Hostname()
+	if err != nil {
+		hostname = "unknown_hostname"
+	} else {
+		hostname = hname
+	}
+
+	ksClient = &http.Client{
+		Timeout: 1 * time.Second,
+	}
+
+	if ksGatewayURL, err = GetKsGatewayUrl(); err != nil {
+		panic(err)
+	}
+}
+
+type KsEvent struct {
+	UUID    string `json:"uuid"`
+	Payload metricspb.Metric `json:"payload"`
```

Comment:
do you mind making `Payload` to be of type:
```
type Payload struct {
    Payload metricspb.Metric `json:"payload"`
    InstanceId string `json:"instance_id"`
}
```
And populate `InstanceId` with `EC2_INSTANCE_ID`? Thank you

Reply:
Added stack and instance id.